main.js

```diff
@@ -0,0 +1,488 @@
+  document.getElementById('turnIndicator').style.display = 'none'
+  document.getElementById('instructions').style.display = 'block'
+  instructionMessage.textContent = 'Enter your name:'
+})
+
+playerVsPlayer.addEventListener('click', function() {
+  document.getElementById('gameType').style.display = 'none'
+  document.getElementById('players').style.display = 'block'
+  document.getElementById('player').style.display = 'none'
+  document.getElementById('turnIndicator').style.display = 'none'
+  document.getElementById('instructions').style.display = 'block'
+  instructionMessage.textContent = 'Enter player names then click Save Names'
+})
+
+playerNamesIn.addEventListener('click', function() {
+  if (xPlayer.value === '') {
+    playerX.name = 'Player X'
+  } else {
+    playerX.name = xPlayer.value
+  }
+  if (oPlayer.value === '') {
+    playerO.name = 'Player O'
+  } else {
+    playerO.name = oPlayer.value
+  }
+  document.getElementById('gameType').style.display = 'none'
+  document.getElementById('players').style.display = 'none'
+  document.getElementById('player').style.display = 'none'
+  document.getElementById('turnIndicator').style.display = 'block'
+  document.getElementById('instructions').style.display = 'block'
+  instructionMessage.textContent = 'Click start button to begin'
+  startUp.removeAttribute('disabled', '')
+})
+
+nameIn.addEventListener('click', function() {
+  if (xPlayer.value === '') {
+    playerX.name = 'Player X'
+  } else {
+    playerX.name = xPlayer.value
+  }
+  playerO.name = 'Dell Latitude 5500'
+  document.getElementById('gameType').style.display = 'none'
+  document.getElementById('players').style.display = 'none'
+  document.getElementById('player').style.display = 'none'
+  document.getElementById('turnIndicator').style.display = 'block'
+  document.getElementById('instructions').style.display = 'block'
+  instructionMessage.textContent = 'Click start button to begin'
+  startUp.removeAttribute('disabled', '')
+})
+
+//     Game play - game live and proceeds by responding to click inputs
+
+// start game - disable start button 
+startUp.addEventListener('click', function() {
+  startUp.setAttribute('disabled', '')
+  enableBoard()
+  document.getElementById('timer').style.display = 'block'
+  instructionMessage.textContent = " Click in a square to place 'X' or 'O' "
+  gameTime()
+  playerX.turn = true
+  playerUp.textContent = playerX.name + ' - PLAY!'
+ })
+
+// add event listeners for board squares
+c0.addEventListener('click', () => {
+  if (cell0.active === true) {
+    if (c0.textContent !== '') {
+      alert('Please select an empty cell.')
+    } else {
+      if (playerX.turn === true) {
+        cell0.owner = 'X'
+        c0.textContent = playerX.token
+      } else {
+        cell0.owner = 'O'
+        c0.textContent = playerO.token
+      }
+      winCalc()
+      if (winCalc() === true) {
+        winner1or2()
+      } else {
+        toggleTurnIndicator()
+      }
+    }
+  }
+})
```

Comment:
Here and below you have some redundant code which can be pulled out into a function, that way you don't have a bunch of repeated code. This will help with readability and organization. One way you could achieve this is by creating an array which contains all of your cells and then iterate over each cell and then add an event listener to each one. Something like this:
```js
let cells = [cell0, cell1, cell2, ... , cell8];
cells.forEach( function(cell) {
  // add an event listener to each cell element
  cell.location.addEventListener( 'click', cellListenerFunction );
})

function cellListenerFunction() {
  // write your repeated code here
  // use event.target to access your cell
  if (event.target.textContent !== "") {
    // alert
  } else {
    if (playerX.turn === true) {
      event.target.textContent = playerX.token
    } 
    // else etc etc
  }
}
```